### ISSUE TITLE
Fix compilation on the FreeBSD

### ISSUE THREAD

**Issue by samm-git** [john30/ebusd#288]
Hi,

I was trying to compile ebusd on freebsd and found that some includes are missing.

Here is a patch:

```
diff --git a/src/lib/ebus/device.h b/src/lib/ebus/device.h
index 7c555f8..115fc36 100755
--- a/src/lib/ebus/device.h
+++ b/src/lib/ebus/device.h
@@ -25,6 +25,7 @@
 #include <netdb.h>
 #include <iostream>
 #include <fstream>
+#include <netinet/in.h>
 #include "lib/ebus/result.h"
 #include "lib/ebus/symbol.h"

diff --git a/src/lib/utils/tcpsocket.h b/src/lib/utils/tcpsocket.h
index 4d0dcb1..fd82b22 100755
--- a/src/lib/utils/tcpsocket.h
+++ b/src/lib/utils/tcpsocket.h
@@ -21,6 +21,7 @@

 #include <unistd.h>
 #include <sys/socket.h>
+#include <netinet/in.h>
 #include <sys/time.h>
 #include <stdint.h>
 #include <string>
```

**Comment by samm-git** [comment]
P.S. i also found that serial port on freebsd is not working, will do another patch soon

**Comment by samm-git** [comment]
Patch added to the https://github.com/john30/ebusd/pull/290

**Comment by john30** [comment]
closed with PR #290
